### DISCUSSION TITLE
I made a real-time captions generator that listens for speech anywhere on your Windows PC

### DISCUSSION THREAD

**Discussion by evermoving** [openai/whisper#2406] [Show and tell]
It records your entire PC audio using WASAPI, allowing real-time captions regardless of which browser, app or game the audio is coming from. The goal was to create a tool that aids language learning and makes digital media more accessible for those who are hard of hearing like myself. 

Accuracy isn't perfect, but it's good. I tried my best to make it detect and be compatible with any speaker/headphone. Feel free to share your feedback. 

https://github.com/evermoving/SystemCaptioner

https://github.com/user-attachments/assets/877db551-48ed-4ca0-8520-c9b2538e3cdf

**Comment by GeorgeLifinRell** [comment]
If it's near real-time, would be perfect!
